cmd/vault-manager/main.go

```diff
@@ -182,13 +182,13 @@ func getConfig() (config, error) {
 		req.Header.Set("Authorization", "Basic "+base64.StdEncoding.EncodeToString([]byte(graphqlUsername+":"+graphqlPassword)))
 	}
 
-	// define a Context for the request
-	ctx := context.Background()
+	ctxTimeout, cancel := context.WithTimeout(context.Background(), time.Second*5)
```

Comment:
I don't fully understand this code, so I won't comment exactly on it. I would expose a fact, though, if it helps. It's known that q-s freezes when reloading the bundle that it has been measured to take up to 6s. q-s liveness tiemouts are set to 10s for that very reason.

Reply:
I appreciate the insight. I suppose it would make sense then to extend the duration to `10s`, for the off chance that this request is made the moment a reload triggers?

Reply:
6s is what @geoberle measured locally using the debugger. I would extend it to 10s as it is what we have done in q-s liveness probes to remove completely the restarts due to failed liveness probe.